tests/src/Mod/Part/App/TopoShapeExpansion.cpp

```diff
@@ -125,30 +123,10 @@ TEST_F(TopoShapeExpansionTest, makeElementCompoundTwoShapesGeneratesMap)
     EXPECT_EQ(4, topoShape.getMappedChildElements().size());  // two vertices and two edges
 }
 
-namespace
-{
-
-std::pair<TopoDS_Shape, TopoDS_Shape> CreateTwoCubes()
-{
-    auto boxMaker1 = BRepPrimAPI_MakeBox(1.0, 1.0, 1.0);
-    boxMaker1.Build();
-    auto box1 = boxMaker1.Shape();
-
-    auto boxMaker2 = BRepPrimAPI_MakeBox(1.0, 1.0, 1.0);
-    boxMaker2.Build();
-    auto box2 = boxMaker2.Shape();
-    auto transform = gp_Trsf();
-    transform.SetTranslation(gp_Pnt(0.0, 0.0, 0.0), gp_Pnt(1.0, 0.0, 0.0));
-    box2.Location(TopLoc_Location(transform));
-
-    return {box1, box2};
-}
-}  // namespace
-
 TEST_F(TopoShapeExpansionTest, makeElementCompoundTwoCubes)
 {
     // Arrange
-    auto [cube1, cube2] = CreateTwoCubes();
+    auto [cube1, cube2] = PartTestHelpers::CreateTwoCubes();
```

Comment:
using namespace PartTestHelpers;

Maybe?  not critical.

Reply:
I have no strong feeling, so I'm happy to change it.